component/src/main/resources/installation-scripts/install.sh

```diff
@@ -14,18 +14,58 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-read -p "Please enter Ballerina home: "  ballerina_home
+ballerina_home=$BALLERINA_HOME
 
 if [ ! -e "$ballerina_home/bin/ballerina" ]
 then
-    echo "Incorrect Ballerina Home provided!"
-    exit 1
+    echo "[WARNING] Unable to find Ballerina home in your system!"
+    read -p "Please enter Ballerina home: "  ballerina_home
+    if [ ! -e "$ballerina_home/bin/ballerina" ]
+    then
+        echo "[ERROR] Incorrect Ballerina Home provided!"
+        exit 1
+    fi
 fi
 
-ballerina_lib_location=$ballerina_home/bre/lib/
-ballerina_balo_location=$ballerina_home/lib/repo/
+ballerina_lib_location=$ballerina_home/bre/lib
+ballerina_balo_location=$ballerina_home/lib/repo
 version=${project.version}
 module_name=kafka
+fileNamePattern="wso2-kafka-module-*.*.*.jar"
+
+for filename in $ballerina_home/bre/lib/*; do
+    existingFile=${filename##*/}
+    [[ $existingFile == $fileNamePattern ]] && file=$existingFile || file=""
+    if [ "$file" != "" ]; then
+        echo "[WARNING] Another version of Kafka module is already installed."
+        read -r -p "Do you want to uninstall it? (Y/N): " response
```

Comment:
Shall we change this to `Do you want to uninstall the previous version and continue installation? (Y/N): `